layouts/_shortcodes/callout.html

```diff
@@ -1,32 +1,57 @@
 {{- /*
 A shortcode to create a callout.
 
-@param {string} type The type of the callout (default, info, warning, error).
+@param {string} type The type of the callout (default, info, warning, error, important).
 @param {string} content The content of the callout.
-@param {string} emoji The emoji of the callout (related to type or can be a custom emoji).
-@param {string} icon The icon of the callout.
+@param {string} emoji The emoji of the callout.
+@param {string} icon The icon of the callout (related to type or can be a custom icon).
 
 @example {{< callout type="info" >}}Content{{< /callout >}}
 */ -}}
 
-{{- $calloutEmojiDict := dict "info" "ℹ️" "warning" "⚠️" "error" "🚫" -}}
 {{- $type := .Get "type" | default "default" -}}
-{{/* If emoji is not set, use the emoji from dict */}}
 {{- $emoji := .Get "emoji" -}}
-{{- if eq $emoji "" -}}
-  {{- $emoji = index $calloutEmojiDict $type -}}
-{{- end -}}
-{{/* Also allow using "icon" */}}
 {{- $icon := .Get "icon" -}}
-{{- $content := .InnerDeindent | markdownify -}}
 
-{{- $defaultClass := "hx:border-orange-100 hx:bg-orange-50 hx:text-orange-800 hx:dark:border-orange-400/30 hx:dark:bg-orange-400/20 hx:dark:text-orange-300" -}}
+{{- $styles := newScratch -}}
```

Comment:
shall we use the same color as [markdown alerts](https://deploy-preview-767--hugo-hextra.netlify.app/docs/guide/markdown/#alerts)?